pkgs/os-specific/linux/tuxedo-drivers/default.nix

```diff
@@ -26,6 +27,9 @@ stdenv.mkDerivation (finalAttrs: {
 
   postInstall = ''
     echo "Running postInstallhook"
+    substituteInPlace usr/lib/udev/rules.d/* \
+      --replace-quiet "/bin/bash" "${lib.getExe pkgsHostHost.bash}" \
```

Comment:
I think `pkgsHostHost` is actually the same as `pkgs` even in a `buildPlatform` != `hostPlatform` scenario?
So this could just be `bash` and that's probably what everybody else is doing?

Reply:
indeed, that is the case